x-pack/plugins/security_solution/server/lib/risk_engine/routes/risk_engine_init_route.ts

```diff
@@ -36,10 +36,7 @@ export const riskEngineInitRoute = (
         if (!taskManager) {
           return siemResponse.error({
             statusCode: 400,
-            body: {
-              message:
-                'Task Manager is unavailable, but is required to initialize the risk engine. Please enable the taskManager plugin and try again.',
-            },
+            body: 'Task Manager is unavailable, but is required to initialize the risk engine. Please enable the taskManager plugin and try again.',
```

Comment:
We didn't used to be able to, but now we can use `i18n` server side - could we extract this to a `i18n` variable to pull in all these places?

Reply:
Thanks for the pointer, implemented in [9693ba1](https://github.com/elastic/kibana/pull/170645/commits/9693ba1ff6d9c25a2b2fd0268ec7aa7802a69bcc)